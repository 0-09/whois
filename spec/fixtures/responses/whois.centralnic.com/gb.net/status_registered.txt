This whois service is provided by CentralNic Ltd and only contains
information pertaining to Internet domain names we have registered for
our customers. By using this service you are agreeing (1) not to use any
information presented here for any purpose other than determining
ownership of domain names, (2) not to store or reproduce this data in 
any way, (3) not to use any high-volume, automated, electronic processes
to obtain data from this service. Abuse of this service is monitored and
actions in contravention of these terms will result in being permanently
blacklisted. All data is (c) CentralNic Ltd https://www.centralnic.com/

Domain ID:CNIC-DO875262
Domain Name:HOTEL.GB.NET
Created On:28-Jun-2012 00:33:28 UTC
Last Updated On:17-Oct-2012 08:51:18 UTC
Expiration Date:28-Jun-2013 23:59:59 UTC
Status:OK
Registrant ID:H2449471
Registrant Name:Alexandra Rodriguez
Registrant Organization:Tiny Spanish
Registrant Street1:36 Britten Close
Registrant City:London
Registrant State/Province:London
Registrant Postal Code:NW11 7HQ
Registrant Country:GB
Registrant Phone:+44.7872039124
Registrant Email:delliks@hotmail.com
Admin ID:H2449471
Admin Name:Alexandra Rodriguez
Admin Organization:Tiny Spanish
Admin Street1:36 Britten Close
Admin City:London
Admin State/Province:London
Admin Postal Code:NW11 7HQ
Admin Country:GB
Admin Phone:+44.7872039124
Admin Email:delliks@hotmail.com
Tech ID:I6467
Tech Name:Domain Admin
Tech Organization:Webfusion
Tech Street1:5 Roundwood Avenue
Tech Street2:Stockley Park
Tech City:Uxbridge
Tech State/Province:Middlesex
Tech Postal Code:UB11 1FF
Tech Country:GB
Tech Phone:+44.8458590018
Tech FAX:+44.8701650437
Tech Email:hosting-domains@webfusion.com
Billing ID:I6467
Billing Name:Domain Admin
Billing Organization:Webfusion
Billing Street1:5 Roundwood Avenue
Billing Street2:Stockley Park
Billing City:Uxbridge
Billing State/Province:Middlesex
Billing Postal Code:UB11 1FF
Billing Country:GB
Billing Phone:+44.8458590018
Billing FAX:+44.8701650437
Billing Email:hosting-domains@webfusion.com
Sponsoring Registrar ID:I6467
Sponsoring Registrar IANA ID:1515
Sponsoring Registrar Organization:Webfusion
Sponsoring Registrar Street1:5 Roundwood Avenue
Sponsoring Registrar Street2:Stockley Park
Sponsoring Registrar City:Uxbridge
Sponsoring Registrar State/Province:Middlesex
Sponsoring Registrar Postal Code:UB11 1FF
Sponsoring Registrar Country:GB
Sponsoring Registrar Phone:0845 859 0018
Sponsoring Registrar FAX:08701 650437
Sponsoring Registrar Website:http://www.123-reg.co.uk/domain-names/
Name Server:NS2.123-REG.CO.UK
Name Server:NS.123-REG.CO.UK
DNSSEC:Unsigned


