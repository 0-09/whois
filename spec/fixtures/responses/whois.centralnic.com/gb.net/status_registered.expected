#referral_whois
  should: %s raise_error(Whois::PropertyNotSupported)

#referral_url
  should: %s raise_error(Whois::PropertyNotSupported)


#status
  should: %s == ["PENDING DELETE", "PENDING DELETE RESTORABLE"]

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s be_a(Time)
  should: %s == Time.parse("2008-12-04 05:56:57 UTC")

#updated_on
  should: %s be_a(Time)
  should: %s == Time.parse("2011-12-04 12:00:12 UTC")

#expires_on
  should: %s be_a(Time)
  should: %s == Time.parse("2011-12-04 23:59:59 UTC")


#nameservers
  should: %s be_a(Array)
  should: %s have(4).items
  should: %s[0] be_a(_nameserver)
  should: %s[0].name == "d.ns14.net"
  should: %s[1] be_a(_nameserver)
  should: %s[1].name == "a.ns14.net"
  should: %s[2] be_a(_nameserver)
  should: %s[2].name == "c.ns14.net"
  should: %s[3] be_a(_nameserver)
  should: %s[3].name == "b.ns14.net"
